python/paddle/v2/fluid/tests/book/test_recognize_digits_mlp.py

```diff
@@ -90,6 +90,17 @@
             fluid.io.save_inference_model(
                 "./recognize_digits_mlp.inference.model/", ["x"], [predict],
                 exe)
+
+            [infer_prog, feed_var_names,
+             fetch_vars] = fluid.io.load_inference_model(
+                 "./recognize_digits_mlp.inference.model/", exe)
+
+            tensor_x = np.random.rand(1, 784).astype("float32")
+            results = exe.run(infer_prog,
+                              feed={feed_var_names[0]: tensor_x},
+                              fetch_list=fetch_vars)
+            print(results[0])
```

Comment:
Can you move the added lines to the end of the file (before exit) and add some comments?

Reply:
Done